bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/WWAToLNuLNuA_4f_NLO/WWAToLNuLNuA_4f_NLO_2/WWAToLNuLNuA_4f_NLO_2_run_card.dat

```diff
@@ -0,0 +1,162 @@
+#***********************************************************************
+ False    = fixed_ren_scale  ! if .true. use fixed ren scale
+ False    = fixed_fac_scale  ! if .true. use fixed fac scale
+ 91.118   = muR_ref_fixed    ! fixed ren reference scale 
+ 91.118   = muF_ref_fixed    ! fixed fact reference scale
+ -1 = dynamical_scale_choice ! Choose one (or more) of the predefined
+           ! dynamical choices. Can be a list; scale choices beyond the
+           ! first are included via reweighting
+ 1.0  = muR_over_ref  ! ratio of current muR over reference muR
+ 1.0  = muF_over_ref  ! ratio of current muF over reference muF
+#*********************************************************************** 
+# Reweight variables for scale dependence and PDF uncertainty          *
+#***********************************************************************
+ 1.0 = rw_rscale ! muR factors to be included by reweighting
+ 1.0 = rw_fscale ! muF factors to be included by reweighting
+ True = reweight_scale ! Reweight to get scale variation using the 
+            ! rw_rscale and rw_fscale factors. Should be a list of 
+            ! booleans of equal length to dynamical_scale_choice to
+            ! specify for which choice to include scale dependence.
+ False = reweight_PDF  ! Reweight to get PDF uncertainty. Should be a
+            ! list booleans of equal length to lhaid to specify for
+            !  which PDF set to include the uncertainties.
+#***********************************************************************
+# Store reweight information in the LHE file for off-line model-       *
+# parameter reweighting at NLO+PS accuracy                             *
+#***********************************************************************
+ True = store_rwgt_info ! Store info for reweighting in LHE file
+#***********************************************************************
+# ickkw parameter:                                                     *
+#   0: No merging                                                      *
+#   3: FxFx Merging - WARNING! Applies merging only at the hard-event  *
+#      level. After showering an MLM-type merging should be applied as *
+#      well. See http://amcatnlo.cern.ch/FxFx_merging.htm for details. *
+#   4: UNLOPS merging (with pythia8 only). No interface from within    *
+#      MG5_aMC available, but available in Pythia8.                    *
+#  -1: NNLL+NLO jet-veto computation. See arxiv:1412.8408 [hep-ph].    *
+#***********************************************************************
+ 0        = ickkw
+#***********************************************************************
+#
+#***********************************************************************
+# BW cutoff (M+/-bwcutoff*Gamma). Determines which resonances are      *
+# written in the LHE event file                                        *
+#***********************************************************************
+ 15000  = bwcutoff
```

Comment:
BW cutoff is set to 15000 for which reason? Sorry I think there were studies on this during one of the GEN meetings. Can you point me the indico links?

Reply:
hi, in the second scheme  p p > ell+ vl ell- vl~ a [QCD] $z, the bwcutoff will define the offshell-onshell Z to suppress zza contribution (and if we exclude all z by "/z", it will induce the divergence of cross-sections and you can check the forth page in slide [https://docs.google.com/presentation/d/1p4a7w0FmeWhdnYRKjtgYKh69es3uNFyrNLUQM_45kQ0/edit?usp=sharing](https://docs.google.com/presentation/d/1p4a7w0FmeWhdnYRKjtgYKh69es3uNFyrNLUQM_45kQ0/edit?usp=sharing) 
 and bwcutoff will not influence the w in this scheme that we refered the bwcutoff explanation here [https://answers.launchpad.net/mg5amcnlo/+faq/2014](https://answers.launchpad.net/mg5amcnlo/+faq/2014) 
we also compared the two different schemes shape in private, you can check the slides in 10 and 11 [https://docs.google.com/presentation/d/1p4a7w0FmeWhdnYRKjtgYKh69es3uNFyrNLUQM_45kQ0/edit?usp=sharing](https://docs.google.com/presentation/d/1p4a7w0FmeWhdnYRKjtgYKh69es3uNFyrNLUQM_45kQ0/edit?usp=sharing) ,and more shape comparison plots here:  [https://zguan.web.cern.ch/zguan/scheme56/](https://zguan.web.cern.ch/zguan/scheme56/) . But in fact, to be on the safe side, we will still only request the official sample of the first scheme, and the second one can be used as our private verification.

Reply:
Thank you for the slides